corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -488,28 +497,42 @@ <h4 class="modal-title">{% trans "Confirm Account" %}</h4>
                       <div class="modal-body">
                         <p class="lead">
                           {% blocktrans %}
-                            Would you like to send the account confirmation email to this account?
+                            Would you like to send the account confirmation {{ 'email' if request|toggle_enabled:"TWO_STAGE_USER_PROVISIONING" else 'sms' }} to this account?
```

Comment:
Is this if/else definitely working? I thought you couldn't use any control flow inside of a blocktrans.

Reply:
Yes, it is not working. Will replace it.